modules/mod_video/support/z_video_convert.erl

```diff
@@ -90,6 +90,7 @@ terminate(_Reason, _State) ->
 code_change(_OldVsn, State, _Extra) ->
     {ok, State}.
 
+-spec do_convert(string(), #state{}) -> ok.
```

Comment:
The spec in line 93 uses a record, please define a type for the record and use that instead.